elink_2_functions.py

```diff
@@ -1,106 +1,128 @@
 # OSTI E-Link 2 documentation https://review.osti.gov/elink2api/
 import requests
-from pprint import pprint
 import mimetypes
 from requests_toolbelt.multipart.encoder import MultipartEncoder
+import cdl_osti_db_functions as cdl
 
 
 # New Metadata submissions
-def submit_new_pubs(pubs_for_metadata_submission, osti_creds):
+def submit_new_pubs(pubs_for_metadata_submission, osti_creds, mysql_creds):
     for pub in pubs_for_metadata_submission:
         print(f"\nSubmitting Publication ID: {pub['id']}")
 
-        response = post_metadata(osti_creds, pub)
-        pub['response_status_code'] = response.status_code
-        pub['response_json'] = response.json()
-        pub['response_success'] = (response.status_code < 300)
+        try:
+            response = post_metadata(osti_creds, pub)
+            pub['response_status_code'] = response.status_code
```

Comment:
You can choose to move response processing to another function to improve code readability.